go/adbc/driver/snowflake/connection.go

```diff
@@ -281,11 +281,14 @@ func (c *cnxn) getObjectsDbSchemas(ctx context.Context, depth adbc.ObjectDepth,
 	}
 
 	conditions := make([]string, 0)
+	queryArgs := make([]interface{}, 0, 2)
 	if catalog != nil && *catalog != "" {
-		conditions = append(conditions, ` CATALOG_NAME LIKE '`+*catalog+`'`)
+		conditions = append(conditions, ` CATALOG_NAME ILIKE ? `)
```

Comment:
The PR comments should indicate that you're also making the conditions case-insensitive.